python-nectar.spec

```diff
@@ -26,24 +27,43 @@ such as the default "threaded" downloader, which uses the "requests" library
 with multiple threads. Other experimental downloaders have used tools like
 pycurl and eventlets.
 
+%package -n python2-nectar
```

Comment:
hardcoded name here is not good, consider to add something like:

%global modname nectar

and

%package -n python2-%{modname}